infermedica_api/webservice.py

```diff
@@ -479,6 +479,69 @@ def risk_factors_list(self):
 
         return models.RiskFactorList.from_json(response)
 
+    def red_flags(self, diagnosis_request, interview_id=None, max_results=8):
+        """
+        Makes an API request with provided diagnosis data and returns a list
+        of observations that may be related to potentially life-threatening
+        conditions.
+
+        :param diagnosis_request: Diagnosis request object or diagnosis json.
+        :type diagnosis_request: :class:`infermedica_api.models.Diagnosis` or dict
+
+        :param interview_id: Unique interview id for diagnosis
+        :type interview_id: str
+
+        :returns: A list of RedFlag objects
+        :rtype: :class:`infermedica_api.models.RedFlagList`
+        """
+        method = self.__get_method('red_flags')
+        headers = self.__get_interview_id_headers(
+            diagnosis_request=diagnosis_request,
+            interview_id=interview_id,
+        )
+
+        request = diagnosis_request
+        if isinstance(diagnosis_request, models.Diagnosis):
+            request = diagnosis_request.get_api_request()
+
+        response = self.__post(
+            method,
+            json.dumps(request),
+            headers=headers,
+            params={'max_results': max_results},
+        )
```

Comment:
I can see you like different styles, but can we be consistent across one code base? :) In all other methods we have __post call in one line (unless it exceed 120 characters), but i guess it will fit.

Reply:
Sure, can do. Thanks!

Reply:
For the consistencies in formatting we might think about setting up a config file and use some formatter like autopep8 as a git hook.